.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly, make sure that your .gitignore file listing  node_modules directory (as it can be added by accident while setting up Linters). When you run" npm init" on the command line, in the root directory of your Node. js module, it will initialize node modules, make sure that any files that are not relevant are ignored by git.